openedx/core/djangoapps/credentials/tests/factories.py

```diff
@@ -26,7 +26,7 @@ class Meta(object):
         model = dict
 
     credential_id = factory.Sequence(lambda n: n)
-    program_id = factory.Sequence(lambda n: n)
+    program_uuid = factory.Sequence(lambda n: n)
```

Comment:
LazyFunction doesn't exist in the version of factory boy edx-platform has but i used LazyAttribute